front/src/styles/scss/applications/operationalStudies/_simulationresults.scss

```
@@ -39,15 +39,24 @@
 
   .osrd-simulation-container {
     position: relative;
-    background-color: white;
     padding: 1rem;
     border-radius: 4px;
 
     &.speedspacechart-container {
+      display: flex;
       background-color: rgb(247, 246, 238);
+      margin-bottom: 28px;
     }
   }
 
+  .simulation-map {
+    position: relative;
+    height: 450px;
+    margin: 0 30px 56px 30px;
```

There should be 48px on the left of the map (10px from global page padding and 38 here)

```suggestion
    margin: 0 30px 56px 38px;
```